resources/install

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/bash
+
+set -e
+
+prefix=$(cat prefix)
+
+appdir="${prefix}/share/applications"
+olddesktop="${appdir}/atom.desktop"
```

Comment:
I would use it if I was going to only rename the desktop file, but I'm also patching it to use a different icon name, so I'll keep it like that for now.